uai2017/python/populate_papers_data.py

```
@@ -0,0 +1,152 @@
+import os
+import sys
+import json
+import argparse
+from openreview import *
+sys.path.append(os.path.join(os.getcwd(), "../../dto/uai2017"))
+print sys.path
+from note_content import *
+from constants import *
+from lxml import etree
+
+"""
+Parse the arguments for user authentication
+"""
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--overwrite', help="If set to true, overwrites existing groups")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+if args.username != None and args.password != None:
+    openreview = Client(baseurl=args.baseurl, username=args.username, password=args.password)
+else:
+    openreview = Client(baseurl=args.baseurl)
+
+
+def get_paper_names(notes):
+    """
+    Getting paper name using the following logic: If note.number is NONE then use 0 as default else 'Paper' followed by its number
+    :param notes:
+    :return:
+    """
+    list_paper_name = []
+    for note in notes:
+        if (note.number is None):
+            list_paper_name.append("Paper0")
```

I think that if the paper doesn't have a number should throw an error